test/test_helpers.py

```diff
@@ -58,8 +56,28 @@ def make_tag(
     type: TagType = TagType.CUMULATIVE_TAIL_TRUNCATED,
     date: Union[pd.Timestamp, str] = "2020-04-02",
     content: str = "taggy",
+    **kwargs,
 ) -> timeseries.TagInTimeseries:
-    return timeseries.TagInTimeseries(type, pd.to_datetime(date), content)
+    date: pd.Timestamp = pd.to_datetime(date)
+    if type == TagType.PROVENANCE:
+        assert content != "taggy"
```

Comment:
What are these checks for?

Reply:
They were leftover from a transition from a previous version of the code. removed.